docs/integrations/sources/hubspot.md

```diff
@@ -57,25 +57,53 @@ Depending on the type of engagement, different properties will be set for that o
 * A `task` engagement will have a corresponding `engagements_metadata` object with non-null values in the `body`, `status`, and `forObjectType` columns.
 
-**Note**: HubSpot API currently only supports `quotes` endpoint using API Key, using Oauth it is impossible to access this stream (as reported by [community.hubspot.com](https://community.hubspot.com/t5/APIs-Integrations/Help-with-using-Feedback-CRM-API-and-Quotes-CRM-API/m-p/449104/highlight/true#M44411)).
+**Note**: HubSpot API currently only supports `quotes` endpoint using API Key, using OAuth it is impossible to access this stream (as reported by [community.hubspot.com](https://community.hubspot.com/t5/APIs-Integrations/Help-with-using-Feedback-CRM-API-and-Quotes-CRM-API/m-p/449104/highlight/true#M44411)).
 
-## Getting Started \(Airbyte Open-Source / Airbyte Cloud\)
+## Getting Started
 
-#### Requirements
+### Requirements \(Airbyte Open-Source\)
 
 * HubSpot Account
-* Api credentials
-* If using Oauth, [scopes](https://legacydocs.hubspot.com/docs/methods/oauth2/initiate-oauth-integration#scopes) enabled for the streams you want to sync
+* API or OAuth2.0 Credentials (See below)
 
-{% hint style="info" %} HubSpot's API will [rate limit](https://developers.hubspot.com/docs/api/usage-details) the amount of records you can sync daily, so make sure that you are on the appropriate plan if you are planning on syncing more than 250,000 records per day. {% endhint %}
+#### Using API Credentials
 
-This connector supports only authentication with API Key. To obtain API key for the account go to settings -&gt; integrations \(under the account banner\) -&gt; api key. If you already have an api key you can use that. Otherwise generated a new one. See [docs](https://knowledge.hubspot.com/integrations/how-do-i-get-my-hubspot-api-key) for more details.
+* API Key
+
+To obtain the API Key for the account, go to settings -&gt; integrations \(under the account banner\) -&gt; API Key. If you already have an API Key you can use that. Otherwise, generate a new one. See [docs](https://knowledge.hubspot.com/integrations/how-do-i-get-my-hubspot-api-key) for more details.
+
+#### Using OAuth2.0 Credentials
+
+* Client ID
+* Client Secret
+* Refresh Token
+* If using OAuth, [scopes](https://legacydocs.hubspot.com/docs/methods/oauth2/initiate-oauth-integration#scopes) enabled for the streams you want to sync
+
+See HubSpot [docs](https://legacydocs.hubspot.com/docs/methods/oauth2/oauth2-quickstart) if you need help finding these fields
+
+{% hint style="info" %}
+HubSpot's API will [rate limit](https://developers.hubspot.com/docs/api/usage-details) the amount of records you can sync daily, so make sure that you are on the appropriate plan if you are planning on syncing more than 250,000 records per day.
+{% endhint %}
+
+### Requirements \(Airbyte Cloud\)
+
+* HubSpot Account
+* Client ID
+* Client Secret
```

Comment:
they don't need any of this oauth stuff in Cloud, they can just click the oauth button. See [this example](https://docs.airbyte.com/integrations/sources/google-ads#getting-started-airbyte-cloud). 

Also could you put Cloud first, for consistency?

Reply:
Replaced with same steps as in the google ads example and moved cloud before OSS